foundation/donate.dd

```diff
@@ -47,6 +47,13 @@ the art in the D programming language.)
 
 $(H3 How can I donate?)
 
+$(DONATE_ITEM Donate through GitHub Sponsors, github,
+
+    The D Language Foundation is enrolled in the GitHub Sponors program. One-time donations here will go toward our general expense fund. We aim to use
```

Comment:
Is it general expense or "general expenses" ?

Reply:
The former. It's similar to how, e.g., "this car costs one-thousand dollars" becomes "it's a one-thousand dollar car".